src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,33 @@
+package duke.task;
+
+/**
+ * Represents an Task object. A <code>Task</code> object
+ * specifies whether it has been completed and its item description.
+ */
+public class Task {
```

Comment:
It might be a good idea to make Task an abstract class, since the concrete instances only consist of Event, Deadline and Todo.